Resources/Prototypes/DeltaV/Entities/Mobs/Cyborgs/borg_chassis.yml

```
@@ -0,0 +1,53 @@
+- type: entity
+  id: BorgChassisSecurity
+  parent: BaseBorgChassisNT
+  name: security cyborg
+  components:
+  - type: Sprite
+    sprite: DeltaV/Mobs/Silicon/chassis.rsi
+    layers:
+    - state: security
+    - state: security_e_r
+      map: ["enum.BorgVisualLayers.Light"]
+      shader: unshaded
+      visible: false
+    - state: security_l
+      shader: unshaded
+      map: ["light"]
+      visible: false
+  - type: BorgChassis
+    maxModules: 4
+    moduleWhitelist:
+      tags:
+      - BorgModuleGeneric
+      - BorgModuleSecurity
+    hasMindState: security_e
+    noMindState: security_e_r
+  - type: Construction
+    node: security
+  - type: IntrinsicRadioReceiver
+  - type: IntrinsicRadioTransmitter
+    channels:
+    - Binary
+    - Common
+    - Science
+    - Security
+  - type: ActiveRadio
+    channels:
+    - Binary
+    - Common
+    - Science
+    - Security
+  - type: AccessReader
+    access: [["Security"], ["Command"], ["Research"]]
+  - type: ShowSecurityIcons
```

```suggestion
  - type: ShowJobIcons
  - type: ShowMindShieldIcons
  - type: ShowCriminalRecordIcons
```
Glad to see this was split up.